shared/guide/system/software/library/package_uuidd_installed.rule

```diff
@@ -0,0 +1,23 @@
+documentation_complete: true
+
+prodtype: ol7
+
+title: 'Package uuidd Installed'
+
+description: |-
+    The package <tt>uuidd</tt> is not installed on Oracle Linux 7 by default. 
```

Comment:
Specific distros should not be called out in ``shared/`` content. Could drop the sentence entirely, adjust to "``is not installed on Linux distributions by default``"..... whatever you think is most appropriate.

(Sorry, did not notice this in original review)